docs/contributing.rst

```diff
@@ -66,6 +66,20 @@ notebook as a base.
 .. _ACCESS-Hive Forum: https://forum.access-hive.org.au/
 
 
+Tips to making a good recipe
+----------------------------
+
+As researchers, we often develop code as part of our research and most of the time the main aim of code development is to find answers to questions that have not yet been explored. Reproducibility may not always be considered as part of the code development process. If this is the case, we are including some tips to writing reproducible code that can be easily followed by other users. 
+
+1. Give your recipe a descriptive but generic title. Avoid being too specific as this will make it easier for other users to find code that matches their needs. For example, ``Computing pairwise distances between grid cells`` may be a better title than ``Computing distance between grid cell and ice edge`` because it does not give users the impression that your recipe is only relevant if they are looking at sea ice.
```

Comment:
```suggestion
1. Give the recipe a descriptive but generic title. Avoid being too specific as this will make it harder for other users to find code that matches their needs. For example, ``Computing pairwise distances between grid cells`` may be a better title than ``Computing distance between grid cells in the Southern Ocean and ice edge`` because the latter gives users the impression that the recipe is only relevant if they are looking at sea ice.
```

as a comment I think we should not call my recipe, your recipe.. but rather out recipe. I tried to use here “the recipe” which is generic.